nx.json

```diff
@@ -45,5 +45,53 @@
       "cache": true
     }
   },
-  "useInferencePlugins": false
+  "useInferencePlugins": false,
+  "pluginsConfig": {
+    "@nx/js": {
+      "analyzeSourceFiles": false
+    }
+  },
+  "release": {
+    "projects": ["packages/**/*"],
+    "changelog": {
+      "workspaceChangelog": {
+        "file": false,
+        "createRelease": "github"
+      }
+    },
+    "version": {
+      "conventionalCommits": false
+    },
+    "conventionalCommits": {
+      "types": {
+        "docs": {
```

Comment:
Looking in commitlintrc, we also have `chore` that could show up in a log. And perhaps `revert` although it would be quite unlikely

I think it's fine to leave out `test` `ci` and `release` because those shouldn't affect users at all and don't need to be listed in changelogs.